### ISSUE TITLE
combinations behave strangely

### ISSUE THREAD

**Issue by dieterdreist** [taginfo/taginfo#178]
I am not sure if this is on purpose, but I have found a strange "coincidence".
Looking at the key site_type, there are 20185 occurences
in combination with historic:civilization there are 1328 (and there are 437 other combinations going down until 1 time, I guess that's because this is a key and not a tag)

Now looking at historic=archaeological_site there are 44806
in combination with site_type there are  17903
but there are no combinations with historic:civilization (and there are only 16 total combinations, it seems there is a cutoff at 1000, because it is a tag)?

While it might be in theory that people tag historic:civilization with site_type and not with historic=archaeological_site, I do find it strange that it appears to be so, I would have expected on the contrary that there would have been more archaeological sites with the civilization tagged but not a site type.

http://taginfo.openstreetmap.org/tags/historic=archaeological_site#combinations
http://taginfo.openstreetmap.org/keys/site_type#combinations

**Comment by joto** [comment]
I am not sure what I am supposed to do with your report. Tags in OSM are often stranger than what most people consider strange, so your hunch that something is strange here doesn't really tell me much. If you think there is a bug in taginfo, then do a count by some other means and compare it.

**Comment by dieterdreist** [comment]
Yes, sorry, I wanted to provide this already at the time I opened the issue but then ran into several timeouts and problems and finally didn't have the time to do it immediately. I have now done this analysis and I think
[historic_civ.osm.zip](https://github.com/joto/taginfo/files/368610/historic_civ.osm.zip)

 it confirms my suspicion.

What have I done:
1. Download in overpass all objects with the key "historic:civilization"
2. count the amount of objects with this tag, there are 4.193
3. count the amount of these which have the tag "historic=archaeological_site": 1.838 objects

Now going back to taginfo, I find these numbers:
2. 4192 (so it's the same)
3. For this I have been looking at historic=archaeological_site in taginfo (44890 total objects) and then went to "combinations" and the 1838 objects with historic:civilization do not show up!

I suspect there might be an issue with the colon ":"

Attached you find the file I used to do the counting (in JOSM)
